test/integration/Integration.Contracts/ItemCommandsTopic.cs

```diff
@@ -8,11 +8,11 @@ public ItemCommandsTopic()
         {
         }
 
-        public string TopicFullName => this.SetTopicName().TopicFullName;
+        public string TopicFullName => this.SetTopicName("ci").TopicFullName;
```

Comment:
hum why is "ci" hardcoded

Reply:
because of testing, will change it to dev.